app/components/Search/ResourcesRow.jsx

```diff
@@ -35,6 +35,55 @@ class ResourcesRow extends Component {
     });
   }
 
+  getOpenTime(schedule) {
+    let date = new Date();
+    let day = date.getDay();
+    let days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+    let hour = date.getHours();
+    let closingOpeningTimes = {};
+    let currDay = days[day];
+    let result = { open: false };
+
+    schedule.forEach(item => {
+      closingOpeningTimes[item.day.replace(/,/g, '')] = {
+        open: item.opens_at,
+        close: item.closes_at
+      };
+    });
+
+    if(closingOpeningTimes[currDay] && hour < closingOpeningTimes[currDay].close) {
+      return {
+        open:true, 
+        time: timeToString(closingOpeningTimes[currDay].close)
```

Comment:
this is the closing time, right? maybe change to something more descriptive like 'closingAt'.